membrane_rtc_engine/mix.exs

```diff
@@ -61,6 +61,9 @@ defmodule Membrane.RTC.Engine.MixProject do
       {:membrane_rtp_format, "~> 0.7.0"},
       {:membrane_opentelemetry, "~> 0.1.0"},
       {:membrane_telemetry_metrics, "~> 0.1.0"},
+      {:membrane_raw_audio_format, "~> 0.11.0"},
+      {:membrane_raw_video_format, "~> 0.3.0"},
+      {:membrane_opus_plugin, "~> 0.17.1"},
```

Comment:
1. Can't we make these deps test-only as well?
2. We're explicitly using a lot of other deps in tests, which aren't specified here -- I think we should add `membrane_h264_(ffmpeg_)plugin`, `membrane_aac_fdk_plugin`, `membrane_rtp_plugin`, `membrane_rtp_h264_plugin` and `membrane_rtp_vp8_plugin` to the test deps... and maybe more, if I missed any :wink:

Reply:
It is not possible to do that because it leads to this beautiful error:
```
Dependencies have diverged:
* membrane_raw_video_format (Hex package)
  the :only option for dependency membrane_raw_video_format

  > In mix.exs:
    {:membrane_raw_video_format, "~> 0.3.0", [env: :prod, hex: "membrane_raw_video_format", only: :test, repo: "hexpm"]}

  does not match the :only option calculated for

  > In deps/membrane_h264_ffmpeg_plugin/mix.exs:
    {:membrane_raw_video_format, "~> 0.3.0", [env: :prod, hex: "membrane_raw_video_format", repo: "hexpm", optional: false]}

  Remove the :only restriction from your dep
```

Reply:
But I will try to add test `:only` to as many dependencies as possible.